### PR TITLE
Remove tokio dep

### PR DESCRIPTION
**Pull request by DavidBM**
Tokio was used only for the main crate. 

Given that we are already using async_std, we can just remove tokio and use async_std function.

This week I though a way to, without changing the zip-rs lib, parallelize the code. I will set this pull request as draft and change the code a bit.

### CODE REVIEW

**Comment by DavidBM** [comment]
Ok. After checking the code doesn't make sense to go deeper into splitting the work in asynchronous tasks as I don't see how it would help. This pull request is ready.

File: src/main.rs [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
     let options = raw_options.process_options();
 
-    for filename in options.files.iter() {
-        let file = File::open(filename).unwrap();
-        let _ = unzip_archive(file, &PathBuf::from(&raw_options.destination_folder)).await;
-    }
-
-    Ok(())
+    task::block_on(async {
```
Comment (elpiel) [inline src/main.rs]: I am not sure which is better:
https://docs.rs/async-std/0.99.11/src/async_std/task/block_on.rs.html#41-102
Or `block_on` on the `futures` crate:
https://github.com/rust-lang-nursery/futures-rs/blob/master/futures-executor/src/local_pool.rs#L60

But I just wanted to put it in here. Looks good to me

File: src/main.rs [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+        for filename in options.files.iter() {
+            let file = File::open(filename).unwrap();
+            let _ = unzip_archive(file, &PathBuf::from(&raw_options.destination_folder));
```
Comment (elpiel) [inline src/main.rs]: Why did you remove the `.await`? how is this even working now :joy_cat:

Comment (DavidBM) [reply src/main.rs]: Ops! I don't know what happened there! Fixed

File: src/main.rs (hunk 1 above)
Comment (DavidBM) [reply src/main.rs]: I think they are basically the same. async_std feels like less magic for me, what helps me to reason about it. But at the end I don't see other benefit than just delete the dependency and reducing the compilation time.

**Comment by DavidBM** [comment]
Ok, answered and fixed. You can check again

File: src/main.rs (hunk 1 above)
Comment (elpiel) [reply src/main.rs]: Makes sense, well if we need it later on we can always change stuff around!
